api/src/api/notes.js

```diff
@@ -12,29 +13,57 @@ const {
   validateReqParams,
 } = require('../middleware/notes');
 
+const replaceIdWithMember = async (ids) => {
+  let memberArray = [];
+  for (let memberId of ids) {
+    const member = await Member.findById(memberId);
+    memberArray.push({
+      memberId: memberId,
+      name: member.firstName + ' ' + member.lastName,
+    });
+  }
+  return memberArray;
+};
+
 router.get(
   '/',
   requireRegistered,
   errorWrap(async (req, res) => {
     let notes;
+    let output_notes = [];
```

Comment:
this can be `const` since we're mutating, but never reassigning. Also can we use camel case here? The snake case (underscores) is sort of random. Maybe `output_notes` could have a better name like `responseNotes` renaming `notes` to `databaseNotes` or something more semantic might be helpful here too.